docs/validators/validator-faq.md

```diff
@@ -10,31 +10,31 @@ This is work in progress. Mechanisms and values are susceptible to change.
+A full node is a program that fully validates transactions and blocks of a blockchain. A full node is distinct from a light node that processes only block headers and a small subset of transactions. Running a full node requires more resources than a light node. A full node is required in order to be a validator. In practice, running a full node implies running a non-compromised and up-to-date version of the software with low network latency and with no downtime.
 
-Of course, it is possible and encouraged for users to run full-nodes even if they do not plan to be validators.
+Of course, it is possible and encouraged for users to run full nodes even if they do not plan to be validators.
 
 ### What is a delegator?
 
-Delegators are Atom holders who cannot, or do not want to run a validator themselves. Atom holders can delegate Atoms to a validator and obtain a part of their revenue in exchange (for more detail on how revenue is distributed, see [**What is the incentive to stake?**](#what-is-the-incentive-to-stake?) and [**What are validators commission?**](#what-are-validators-commission?) sections below).
+Delegators are ATOM holders who cannot, or do not want to, run a validator themselves. ATOM holders can delegate ATOM to a validator and obtain a part of their revenue in exchange. For details on how revenue is distributed, see [What is the incentive to stake?](#what-is-the-incentive-to-stake?) and [What are validators commission?](#what-are-validators-commission?) in this document.
 
-Because they share revenue with their validators, delegators also share risks. Should a validator misbehave, each of their delegators will be partially slashed in proportion to their delegated stake. This is why delegators should perform due diligence on validators before delegating, as well as spreading their stake over multiple validators.
+Because delegators share revenue with their validators, they also share risks. If a validator misbehaves, each of their delegators are partially slashed in proportion to their delegated stake. This penalty is one of the reasons why delegators must perform due diligence on validators before delegating. Spreading their stake over multiple validators is another layer of protection.
 
-Delegators play a critical role in the system, as they are responsible for choosing validators. Being a delegator is not a passive role: Delegators should actively monitor the actions of their validators and participate in governance. For more, read the [delegator's faq](https://hub.cosmos.network/main/delegators/delegator-faq.html).
+Delegators play a critical role in the system, as they are responsible for choosing validators. Being a delegator is not a passive role. Delegators must actively monitor the actions of their validators and participate in governance. For details on being a delegator, read the [Delegator FAQ](https://hub.cosmos.network/main/delegators/delegator-faq.html).
```

Comment:
Being a delegator is not a passive role. &#8594; Delegator is not a passive role

Reply:
i think both make sense, but barrie's sounds more natural to my ear. let's stay with her suggestion